python/core/auto_generated/qgsclipper.sip.in

```diff
@@ -66,7 +69,35 @@ an open shape (linestring).
 
 %End
 
-    static void trimPolygon( QPolygonF &pts, const QgsRectangle &clipRect );
+    static void trimPolygon( QPolygonF &polygon, const QgsRectangle &clipRect );
```

Comment:
```suggestion
    static void trimPolygon( QPolygonF &pts, const QgsRectangle &clipRect );
```

We can't change argument names in existing methods, as it would break python code which use named arguments